test/conformance/tests/test_sriov_operator.go

```diff
@@ -1368,6 +1368,125 @@ var _ = Describe("[sriov] operator", func() {
 					Expect(stdout).To(ContainSubstring("2 packets transmitted, 2 received, 0% packet loss"))
 				})
 			})
+
+			Context("ExcludeTopology", func() {
+				BeforeEach(func() {
+					if discovery.Enabled() {
+						Skip("Test unsuitable to be run in discovery mode")
+					}
+				})
+
+				It("field is forwarded to the device plugin configuration", func() {
+					node := sriovInfos.Nodes[0]
+					sriovDeviceList, err := sriovInfos.FindSriovDevices(node)
+					Expect(err).ToNot(HaveOccurred())
+					unusedSriovDevices, err := findUnusedSriovDevices(node, sriovDeviceList)
+					Expect(err).ToNot(HaveOccurred())
+
+					intf := unusedSriovDevices[0]
+
+					excludeTopologyTrue := &sriovv1.SriovNetworkNodePolicy{
```

Comment:
`excludeTopologyTrue`, `excludeTopologyFalse` can be moved to common place under Context ?

i see they both defined the same in both test cases.

Reply:
I moved the declaration under Context. BTW, assignments need the node name and the interface name, so I had to put them under BeforeEach. WDYT?